### ISSUE TITLE
Integration with CSV Import

### ISSUE THREAD

**Issue by sharonmleon** [omeka-s-modules/CustomVocab#22]
It would be nice to have the custom vocabs available as data types for properties in the mapping section of CSV Import. It is possible to update the data type later with data cleaning, but it would be more efficient if the import could handle it.

**Comment by jimsafley** [comment]
I've added this feature in the [csv-import-data-types branch](https://github.com/omeka-s-modules/CustomVocab/compare/csv-import-data-types). This change depends on a merged [CSVImport PR](https://github.com/omeka-s-modules/CSVImport/pull/194), so we need to wait for that to be released before we can release this.
